agent/consul/fsm/commands_oss.go

```diff
@@ -260,24 +260,7 @@ func (c *FSM) applyACLOperation(buf []byte, index uint64) interface{} {
 			return err
 		}
 		return enabled
-	case structs.ACLBootstrapNow:
```

Comment:
Should we leave this case in place so we can create a reasonable error message during raft replay?

Reply:
Ya, that sounds good. I'll do the same for `structs.ACLForceSet`, but I'll probably inline the values so we can still remove the old constants.